include/sys/boardctl.h

```diff
@@ -392,6 +393,17 @@ struct boardioc_nxterm_ioctl_s
 };
 #endif /* CONFIG_NXTERM */
 
+#ifdef CONFIG_BOARDCTL_BOOT_IMAGE
+
+/* Structure containing the arguments to the BOARDIOC_BOOT_IMAGE command */
+
+struct boardioc_boot_info_s
+{
+  FAR const char *img_path;           /* Path to application image */
+  uint32_t        img_header_size;    /* Size of the image header in bytes */
```

Comment:
The header is defined by mcuboot or vendor?

Reply:
This is from MCUboot:
https://github.com/mcu-tools/mcuboot/blob/main/boot/bootutil/include/bootutil/image.h#L129-L138

The vendor may also add a specific header, but that will be part of the image payload.

Reply:
Ok.

Reply:
how about remove img_ prefix? It look like the redundant.

Reply:
> Ok.

Good, I'll rename every reference of **application image** to **application firmware image** then.

> how about remove img_ prefix? It look like the redundant.

Ok, seems reasonable. I'll do it.

Reply:
@xiaoxiang781216 Done, please take another look.